hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/PrefetchExecutor.java

```diff
@@ -44,12 +50,15 @@ public final class PrefetchExecutor {
 
   /** Futures for tracking block prefetch activity */
   private static final Map<Path, Future<?>> prefetchFutures = new ConcurrentSkipListMap<>();
+  /** Set of files for which prefetch is completed */
+  public static HashMap<String, Boolean> prefetchCompleted = new HashMap<>();
```

Comment:
nit: do we really need a map here? It seems we never check the boolean value, but only if the map has an entry for the path of not. Could we use hashset or a list impl?

Reply:
Thanks for the review Wellington :) 
Yes, my initial approach was to use a set. But I could find a protobuf support for HashSet.

Also, I wanted to maintain a unique set of filenames so I could avoid doing a duplicate check. So I chose a map implementation. 

https://developers.google.com/protocol-buffers/docs/reference/java-generated#map-fields